### ISSUE TITLE
Detect keywords rather than any text

### ISSUE THREAD

**Issue by JerryLife** [izhangzhihao/intellij-rainbow-fart#27]
She speaks when I type ```CIFAR10``` or ```forward``` because of recognizing ```IF``` or ```for```. That shouldn't have happened. Only the keywords rather than any text should be able to trigger the voice.

**Comment by izhangzhihao** [comment]
Right! The keyword should only triggered by the code rather not comments/variable names or something else
